airbyte-ci/connectors/ci_credentials/pyproject.toml

```diff
@@ -11,7 +11,7 @@ readme = "README.md"
 packages = [{ include = "ci_credentials" }]
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "^3.11"
```

Comment:
Was going back through this pr. I think there is some bug with poetry because I have seen sometimes the carat doesn't get respected as we would expect, and I had to explicitly put 3.11 to get it to work